csharp/ql/test/library-tests/dataflow/global/GlobalDataFlow.cs

```diff
@@ -231,6 +232,21 @@ public void M2()
         Check(nonSink);
     }
 
+    public async void M3()
+    {
+        // async await, tainted
+        var sink41 = Task.Run(() => "taint source");
+        Check(sink41);
+        var sink42 = await sink41;
```

Comment:
Rename to `sink1`?

Reply:
`sink0` and `sink1` are already used in another test, so that's why I went for the next available numbers.

Reply:
Hmm, okay!